src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEFieldSymbol.cs

```diff
@@ -20,6 +20,56 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols.Metadata.PE
     /// </summary>
     internal sealed class PEFieldSymbol : FieldSymbol
     {
+        private struct PackedFlags
+        {
+            // Layout:
+            // |..............................|vvvvv|
+            //
+            // f = FlowAnalysisAnnotations. 5 bits (4 value bits + 1 completion bit).
+
+            private const int HasDisallowNullAttribute = 0x1 << 0;
+            private const int HasAllowNullAttribute = 0x1 << 1;
+            private const int HasMaybeNullAttribute = 0x1 << 2;
+            private const int HasNotNullAttribute = 0x1 << 3;
+            private const int FlowAnalysisAnnotationsCompletionBit = 0x1 << 4;
+
+            private int _bits;
+
+#if DEBUG
+            static PackedFlags()
+            {
+                Debug.Assert(EnumUtilities.ContainsAllValues<FlowAnalysisAnnotations>(0xFF));
```

Comment:
The assert seems unrelated to the number of bits stored.

Reply:
I'd kept the assertion to alert us if the enum changes, so that we consider adding handling here. That said, that seems superfluous, so I'll remove.

---
In reply to: [296972213](https://github.com/dotnet/roslyn/pull/36704#discussion_r296972213) [](ancestors = 296972213)